aiida/tools/importexport/migration/__init__.py

```diff
@@ -52,8 +52,8 @@ def migrate_recursively(metadata, data, folder, version=EXPORT_VERSION):
 
     try:
         if old_version == version:
-            raise ArchiveMigrationError('Your export file is already at the version {}'.format(version))
-        elif old_version > version:
+            return old_version
```

Comment:
This is changing the user-expected behaviour of the migration functionality, something quite out-of-scope of this PR.

Reply:
While I would consider the new behavior the expected one, I agree that I am changing it here and that there was no issue opened for it where it was discussed. I will open it now.

Reply:
Opened #4322